scripts/cmake/ElektraCache.cmake

```diff
@@ -353,7 +353,8 @@ set (
 	""
 	CACHE STRING "Optional suffix to use on lib folders (e.g. 64 for lib64)")
 
-set (MEMORYCHECK_COMMAND_OPTIONS "--trace-children=yes --leak-check=full --gen-suppressions=all --error-exitcode=1")
+set (MEMORYCHECK_COMMAND_OPTIONS
+     "--trace-children=yes --leak-check=full --show-leak-kinds=all --track-origins=yes --gen-suppressions=all --error-exitcode=1")
```

Comment:
@petermax2 maybe you can use this to show which leaks are detected in crypto?

Reply:
We would probably need the debug symbols of gpg nevertheless.

Reply:
I am working on adding debug symbols for some of the used libraries. 👍